File: test/fixtures/node-got/post_with_extra_whitespace.js
import got from 'got';
import * as fs from 'fs';

const form = new FormData();
form.append('files', fs.readFileSync('47.htz'), '47.htz');
form.append('name', '47');
form.append('oldMediaId', '47');
form.append('updateInLayouts', '1');
form.append('deleteOldRevisions', '1');

const response = await got.post('http://localhost:28139/api/library', {
  headers: {
    'accept': 'application/json'
  },
  body: form
});

File: test/fixtures/ruby-httparty/post_with_extra_whitespace.rb
require 'httparty'

url = 'http://localhost:28139/api/library'
headers = {
  'accept': 'application/json',
  'Content-Type': 'multipart/form-data',
}
body = {
  files: File.open('47.htz')
  name: '47'
  oldMediaId: '47'
  updateInLayouts: '1'
  deleteOldRevisions: '1'
}
res = HTTParty.post(url, headers: headers, body: body)
